build/generate-docs.ts

```diff
@@ -166,6 +166,7 @@ function typeToMarkdownLink(type: string): string {
         case 'promoteid':
             return ` [${type}](types.md)`;
         case 'color':
+        case 'projection-mode':
```

Comment:
"projection" is already used for the top-level category, and we need a value type for the projection expression, and it has to have a different name than the top level to avoid recursion issues.

Reply:
Top level
https://github.com/maplibre/maplibre-style-spec/pull/888/files#diff-d9c8777c5eac8bd8dadac08a9fdc12df4109935355f2c98613b191436125a09bR129